src/dynarec/la64/dynarec_la64_660f.c

```diff
@@ -528,6 +528,32 @@ uintptr_t dynarec64_660F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
         case 0x3A: // these are some more SSSE3+ opcodes
             opcode = F8;
             switch (opcode) {
+                case 0x0B:
+                    INST_NAME("ROUNDSD Gx, Ex, Ib");
+                    nextop = F8;
+                    GETGX(q0, 1);
+                    GETEXSD(q1, 0, 1);
+                    u8 = F8;
+                    v0 = fpu_get_scratch(dyn);
+                    v1 = fpu_get_scratch(dyn);
+                    XVREPLVE0_D(v0, q1);
```

Comment:
What is this for? I don't not follow.

Reply:
copy src lo64 to tmp every elements.

Reply:
Yeah I know what this instruction do, but what is it for?